MCSync/frmMain.cs

```diff
@@ -14,95 +23,77 @@ public frmMain()
             InitializeComponent();
         }
 
-        private void cmdConfig_Click(object sender, EventArgs e)
-        {
-            frmConfig config = new frmConfig();
-            config.ShowDialog();
-        }
-
-        private void cmdSync_Click(object sender, EventArgs e)
-        {
-            frmSync sync = new frmSync();
-            sync.Show();
-            Hide();
-        }
         private void frmMain_Load(object sender, EventArgs e)
         {
-            lblDocumentation.Visible = false;
             backgroundChooser();
+            lblVersion.Text = "Version " + Application.ProductVersion;
+            pnlMainTimer.Start();
         }
 
-        private void newBtnConf_MouseEnter(object sender, EventArgs e)
-        {
-            btnConf.BackgroundImage = Properties.Resources.btnOptOver;
-        }
-
-        private void newBtnConf_MouseLeave(object sender, EventArgs e)
+        public void backgroundChooser()
         {
-            btnConf.BackgroundImage = Properties.Resources.btnOptBase;
-        }
+            try
+            {
+                int rand = new Random().Next(1, 105);
 
-        private void newBtnSync_MouseEnter(object sender, EventArgs e)
-        {
-            btnSync.BackgroundImage = Properties.Resources.btnSyncOver;
-        }
+                var request = WebRequest.Create("https://mesabrook.com/backgrounds/background" + rand + ".png");
 
-        private void newBtnSync_MouseLeave(object sender, EventArgs e)
-        {
-            btnSync.BackgroundImage = Properties.Resources.btnSyncBase;
+                using (var response = request.GetResponse())
+                using (var stream = response.GetResponseStream())
+                {
+                    BackgroundImage = Bitmap.FromStream(stream);
+                    BackgroundImageLayout = ImageLayout.Stretch;
+                }
+            }
+            catch (Exception ex)
+            {
+                BackgroundImage = Properties.Resources.b1;
+            }
         }
 
-        private void newBtnConf_MouseClick(object sender, MouseEventArgs e)
+        private void fancyButton1_Click(object sender, EventArgs e)
         {
+            pnlMain.Hide();
             frmConfig config = new frmConfig();
-            config.ShowDialog();
-        }
 
-        private void newBtnSync_MouseClick(object sender, MouseEventArgs e)
-        {
-            frmSync sync = new frmSync();
-            sync.Show();
-            Hide();
+            config.TopLevel = false;
+            config.ControlBox = false;
+            config.Dock = DockStyle.Fill;
+            config.FormBorderStyle = FormBorderStyle.None;
```

Comment:
Should these options be set on the form itself rather than manually here?

Reply:
Fixed